agentforce/1.1/modules/ROOT/nav.adoc

```diff
@@ -0,0 +1,7 @@
+.xref:index.adoc[Agentforce Connector]
+* xref:index.adoc[Agentforce Connector Overview]
+* xref:agentforce-connector-config-acb.adoc[Using Anypoint Code Builder to Configure Agentforce Connector]
+* xref:agentforce-connector-studio.adoc[Using Anypoint Studio to Configure Agentforce Connector]
+* xref:configuring-agent-operations.adoc[]
```

Comment:
I'd hardcode this link in both 1.0 and 1.1 navs because now since there are two versions, the version numbers are present 

* xref:configuring-agent-operations.adoc[Configuring Agent Operations for Agentforce Connector]